pom.xml

```diff
@@ -45,62 +52,26 @@
   </dependencies>
 
   <build>
-    <pluginManagement>
-      <plugins>
-        <plugin>
-          <artifactId>maven-compiler-plugin</artifactId>
-          <version>3.5.1</version>
-          <configuration>
-            <source>1.8</source>
-            <target>1.8</target>
-          </configuration>
-        </plugin>
-      </plugins>
-    </pluginManagement>
-
     <plugins>
+
       <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-shade-plugin</artifactId>
-        <version>2.4.3</version>
+        <groupId>io.reactiverse</groupId>
+        <artifactId>vertx-maven-plugin</artifactId>
+        <version>1.0.17</version>
         <executions>
           <execution>
-            <phase>package</phase>
+            <id>vmp</id>
             <goals>
-              <goal>shade</goal>
+              <goal>package</goal>
```

Comment:
It should be:
```
<plugin>
            <groupId>io.reactiverse</groupId>
            <artifactId>vertx-maven-plugin</artifactId>
            <version>${version}</version>
            <executions>
                <execution>
                    <id>vmp</id>
                    <goals>
                        <goal>initialize</goal>
                        <goal>package</goal>
                    </goals>
                </execution>
            </executions>
            <configuration>
                <redeploy>true</redeploy>
            </configuration>
        </plugin>
```

Reply:
With the right version of course.

Reply:
I removed the initialize goal because it's only applicable for projects with js / webjar dependencies. However, if you think that should be the default then I'll add it back in. :+1: 

> This is the default configuration. It builds the fat jar in the package phase, and support .js and webjars dependencies. mvn compile vertx:run enables the redeployment.

https://reactiverse.io/vertx-maven-plugin/#package-goal-examples
https://reactiverse.io/vertx-maven-plugin/#vertx:initialize